src/core/serializedtypes.js

```diff
@@ -258,8 +258,8 @@ function parse(so) {
 
   const field_bits = tag_byte & 0x0f;
   let field_name = (field_bits === 0)
-    ? field_name = binformat.fields[type_bits][so.read(1)[0]]
```

Comment:
is the syntax correct? filed_name = ( test ? expression1 : expression2 )
if (field_bits===0) assign the value of line 261 to field_name otherwise line 262